### PR TITLE
kernel-build.eclass: handle CTF if enabled

### PR DESCRIPTION
**Pull request by thesamesam**
genpatches will soon be adding optional CTF debug information support via a patch. If it's enabled, we need to call a few special make targets to create and install the CTF information. We also need to avoid stripping the CTF archive.

### CODE REVIEW

**Comment by thesamesam** [comment]
cc @gentoo/dist-kernel

**Review by mgorny — APPROVED** [review]
LGTM, presuming you've tested it.

File: eclass/kernel-build.eclass [hunk 1]
```diff
@@ -337,6 +343,10 @@ kernel-build_src_install() {
 		targets+=( dtbs_install )
 	fi
 
+	if grep -q "CONFIG_CTF=y" "${WORKDIR}/modprep/.config"; then
+		targets+=( ctf_install )
```
Comment (AndrewAmmerlaan) [inline eclass/kernel-build.eclass:347]: Should we also `ctf_install` in `src_test`?

Comment (thesamesam) [reply eclass/kernel-build.eclass:347]: Good catch. It shouldn't matter much but it would be nice to ensure the target works correctly. I'll add that in.

Comment (thesamesam) [reply eclass/kernel-build.eclass:347]: Done in 983fe336a7571f822655c25e0785f067de80c64c.

**Pull request merged**